src/main/java/com/shopify/ShopifySdk.java

```diff
@@ -526,6 +527,9 @@ public ShopifyInventoryLevel updateInventoryLevel(final String inventoryItemId,
 	public ShopifyRefund refund(final ShopifyRefundCreationRequest shopifyRefundCreationRequest) {
 
 		final ShopifyRefund calculatedShopifyRefund = calculateRefund(shopifyRefundCreationRequest);
+		calculatedShopifyRefund.getTransactions().forEach(transaction -> {
```

Comment:
Please remove curly braces here to fix SonarCloud violation:
https://sonarcloud.io/project/issues?branch=feature%2FRefunds&id=com.channelape%3Ashopify-sdk&open=AWblcT5_SgN4uCclg3RP&resolved=false